source/_integrations/hue.markdown

```diff
@@ -36,20 +36,37 @@ There is currently support for the following device types within Home Assistant:
+In the Hue concept you can create (dynamic) scenes for the lights within rooms and zones. You can create, edit and delete Hue scenes from the (official) Hue app on iOS and Android. Each Zone/Room can have it's own scenes assigned and there is a large library of precreated scenes for specific moods. These Hue scenes are automatically imported in Home Assistant and they're available as `scene entities`. Creating or editing Hue scenes in Home Assistant is not supported.
 
+`It is advised to use Hue scenes for controlling multiple lights at once for a smooth experience. If you individually control multiple lights and/or use Home Assistant scenes, each light command will be sent to each light one by one which doesn't give a very good user experience, while using a Hue scene sends commands to all lights at once in an optimized way, resulting in a smooth experience.`
+
+### Service `hue.activate_scene`
+
+To have more control over Hue scenes we've implemented a secondary, more advanced service to activate a Hue scene and set some properties at the same time, such as the Dynamic mode and/or brightness.
+
+| Service Data Attribute | Required | Description                                                                                   |
+| ---------------------- | -------- | --------------------------------------------------------------------------------------------- |
+| `entity_id`            | yes      | Entity ID of the Hue Scene entity you want to activate.                                       |
+| `transition`           | no       | Transition duration (in seconds) it takes to bring devices to the state defined in the scene. |
+| `dynamic`              | no       | Enable (true) or Disable (false) dynamic mode for the scene.                                  |
+| `speed`                | no       | Set the speed (of the dynamic palette) for this scene.                                        |
+| `brightness`           | no       | Set the brightnes for this scene.                                                             |
+
+You can use this service for example if you'd like to start/stop Dynamic Mode.
 
 ## Hue remotes and switches
 
 Hue remotes such as the Dimmer Switch are stateless devices, meaning that they do not have a on/off state like regular entities in Home Assistant. Instead, such devices emit the event `hue_event` when a button is pressed. You can test what events come in using the event {% my developer_events title="developer tools in Home Assistant" %} and subscribe to the `hue_event`. Once you know what the event data looks like, you can use this to create automations.
 
+`NOTE: At time of writing there's a limitation on the Hue API that eacht device can only send one event per second. This means that button events are rate limited to 1 per second. This is brought to the attention of Signify and it will hopefully be fixed soon.`
```

Comment:
This is not code.

```suggestion
<div class='note warning'>

At the time of writing, there's a limitation on the Hue API that each device can only send one event per second. This means that button events are rate-limited to 1 per second. This is brought to the attention of Signify and it will hopefully be fixed soon.

</div>
```

Reply:
That is the one I was looking for.